### DISCUSSION TITLE
Problème lors de l'installation chamilo (étape 2/6)

### DISCUSSION THREAD

**Discussion by flxr-m** [chamilo/chamilo-lms#5164] [General]
Bonjour, 
Je tente d'installer chamilo (version 1.11.26) sur un serveur ubuntu (version 20.04). J'ai installé et configuré correctement apache 2.4 (redirections OK), php7.4, LAMP et mysql 8.0. Lorsque j'arrive à l'étape 2/6 de l'installation, certaines extensions php sont affichées en rouge, je les installe, redémarre le serveur et cela se met ensuite en vert. Jusque là pas de problème, mais un petit peu en dessous, certains paramètres du fichier de configuration php.ini (situé dans /etc/php/7.4/apache2/php.ini) sont affichés en rouge. Je les corrige, je redémarre le serveur, mais cela ne fait rien, rien ne change, c'est toujours affiché en rouge. Je ne parviens pas à trouver le problème, que puis-je faire ? 

Merci

**Comment by ywarnier** [comment]
Bonjour @flxr-m 
Parfois les paramètres php ne sont pas vraiment où on le pense.
Je te conseille de d'abord placer (temporairement jusqu'à résoudre le problème) un fichier "info.php" à la racine de ton Chamilo et à le charger avec ton navigateur. Il doit seulement contenir ceci:
```
<?php
phpinfo();
```
Ça te permettra de voir l'état des paramètres sans intervention de Chamilo, et sans intermédiaire. Tu peux donc recharger cette page dès que tu fais une modification de php.ini et voir s'il y a une différence (tu peux jouer, par exemple, avec  `max_execution_time` pour voir si la valeur que tu lui donnes est bien prise en compte.

Si elle n'est pas prise en compte, c'est probablement parce qu'il y a un autre fichier php.ini quelque part qui écrase les valeurs que tu définis dans ton php.ini.
